frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -787,3 +686,205 @@ export function CodeLanguageSelect(): JSX.Element {
         </Select>
     )
 }
+
+interface ExperimentPreviewProps {
+    experiment: Partial<any> | null
+    trendCount: number
+    exposure: number
+    conversionRate: number
+    runningTime: number
+    sampleSize: number
+}
+
+export function ExperimentPreview({
+    experiment,
+    trendCount,
+    exposure,
+    conversionRate,
+    runningTime,
+    sampleSize,
+}: ExperimentPreviewProps): JSX.Element {
+    const { experimentInsightType, experimentId } = useValues(experimentLogic)
+    const [currentVariant, setCurrentVariant] = useState('control')
+
+    return (
+        <Card className="experiment-preview">
+            <Row>
+                <Col span={experimentId === 'new' ? 24 : 12}>
+                    <Row className="experiment-preview-row">
+                        <Col>
+                            <div className="card-secondary mb-05">Preview</div>
+                            <div>
+                                <span className="mr-05">
+                                    <b>{experiment?.name}</b>
+                                </span>
+                                {experiment?.feature_flag_key && (
+                                    <CopyToClipboardInline
+                                        explicitValue={experiment.feature_flag_key}
+                                        iconStyle={{ color: 'var(--text-muted-alt)' }}
+                                        description="feature flag key"
+                                    >
+                                        <span className="text-muted">{experiment.feature_flag_key}</span>
+                                    </CopyToClipboardInline>
+                                )}
+                            </div>
+                        </Col>
+                    </Row>
+                    <Row className="experiment-preview-row">
+                        {experimentInsightType === InsightType.TRENDS ? (
+                            <>
+                                <Col span={12}>
+                                    <div className="card-secondary">Baseline Count</div>
```

Comment:
this kind of calculation seems to have introduced subtle bugs into experiments:

1. On the preview page, these calculations happen thanks to the funnel that existed *at that moment in time*
2.  On the results page, instead of doing these calculations again (which generate a new insight from present day & get the numbers from it) - we get these values from what's stored on the experiment: the `recommended_sample_size` or `recommended_running_time` parameters.

(We don't store baseline count, so that doesn't show up in the preview on results).

Reply:
Ahh yeah I think it's just starting to get confusing because we have so many variables we need floating around the frontend, since those "creation" values need to show up on the results page too and aren't strictly shown just during creation, right? But thanks for catching this and let's definitely try to get this to a state where we can easily compile and move the variables out of the frontend 😆

Reply:
yeahhh, it's a shame: I wanted to use the selectors directly, but this approach wasn't viable: it was getting to hard to get the results of the currently shown funnel into a selector, which is why I decided to put variables in there.

(think there's an old PR up for this, and twixes pitched in saying the same, so 🤷‍♂️ )

Reply:
(I started some work to move significance calculations to the backend as well, so that should clean some things up - convinced we wouldn't need significance value sliders on the results page, as no user interview has brought that up)